bot/resources/tags/decorators.md

```diff
@@ -2,7 +2,23 @@
 
```

Comment:
The branch is a bit out of date, can you rebase onto main/merge this please ?

Reply:
Done